### PR TITLE
INFODEV-232 [MVP] Rackspace Migration Manager: Add new How-To article with the products, flavors, and operating systems

### PR DESCRIPTION
**Pull request by cyrichardson**
Supported by Rackspace Migration Manager.

Used wiki information provided by Amit:
Scope – What products & features will migrate from RAX to AWS at launch (EX: Servers, Networks only // Servers, Networking, DBaaS, etc…) 
Products: https://one.rackspace.com/display/OPS/Rackspace+Migration+Manager+-+Project+Plan
Flavors: https://one.rackspace.com/display/OPS/MT+-+Flavors+Supported
Operating Systems: https://one.rackspace.com/display/OPS/MT+-+PV+to+HVM

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-69d52bc946/how-to/). :bow:

**Comment by cyrichardson** [comment]
@amitgandhinz @th3incredibl3hulk  Please review this new article when you have time. Thanks!

**Comment by cyrichardson** [comment]
Also @McNinja - please review.

File: content/rackspace-migration-manager/scope.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+type: article
+created_date: '2017-08-09'
+created_by: Catherine Richardson
+last_modified_date: '2017-08-09'
+last_modified_by: Catherine Richardson
+product: Rackspace Migration Manager
+product_url: rackspace-migration-manager
+---
+
+The initial launch of Rackspace Migration Manager includes support for
```
Comment (sameerkotasthane) [inline content/rackspace-migration-manager/scope.md]: test

**Comment by cyrichardson** [comment]
Thanks, @McNinja!

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a1f73c282b/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-10f1a1a75b/how-to/)

**Comment by cyrichardson** [comment]
Hi, @kmsholcomb! I think you can go ahead and edit this one when you have a chance. It's mostly tables and lists. I opened a JIRA task for it -  https://jira.rax.io/browse/INFODEV-233.

**Comment by kmsholcomb** [comment]
Hi @cyrichardson. Edits are attached. Please let me know if you have any questions. Thanks!

[Product flavor and OS support in Rackspace Migration Manager_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1218497/Product.flavor.and.OS.support.in.Rackspace.Migration.Manager_editsKH.docx)
[Product flavor and OS support in Rackspace Migration Manager_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1218496/Product.flavor.and.OS.support.in.Rackspace.Migration.Manager_editsKH.pdf)

**Comment by cyrichardson** [comment]
@kmsholcomb Thank you!

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-91e9dd1c34/how-to/)

**Comment by cyrichardson** [comment]
Closing in favor of https://jira.rax.io/browse/INFODEV-424

**Pull request closed without merging**